zsh/_brew-koji

```diff
@@ -43,482 +121,1268 @@ _brew-koji() {
          'list-targets:List the build targets'
          'list-tasks:Print the list of tasks'
          'list-untagged:List untagged builds'
-         'maven-build:Build a Maven package from source'
-         'maven-chain:Run a set of Maven builds in dependency order'
+         'list-volumes:List storage volumes'
          'mock-config:Create a mock config'
-         'move-pkg:Move one or more packages between tags'
-         'resubmit:Retry a canceled or failed task, using the same parameter as the original task.'
          'rpminfo:Print basic information about an RPM'
-         'search:Search koji'
          'show-groups:Show groups data for a tag'
-         'tag-pkg:Apply a tag to one or more packages'
          'taginfo:Print basic information about a tag'
          'taskinfo:Show information about a task'
-         'untag-pkg:Remove a tag from one or more packages'
+         'call:Execute an arbitrary XML-RPC call'
+         'dist-repo:Create a yum repo with distribution options'
+         'import-comps:Import group/package information from a comps file'
+         'moshimoshi:Introduce yourself'
+         'add-notification:Add users notification'
+         'block-notification:Block users notifications'
+         'edit-notification:Edit users notification'
+         'list-notifications:List users notifications and blocks'
+         'remove-notification:Remove users notifications'
+         'unblock-notification:Unblock users notification'
          'wait-repo:Wait for a repo to be regenerated'
          'watch-logs:Watch logs in realtime'
          'watch-task:Track progress of particular tasks'
-         'win-build:Build a Windows package from source'
+         'search:Search the system'
       )
       _describe -t commands 'command' commands && ret=0
       ;;
     (args)
       case $words[1] in
+       (add-external-repo)
+         _arguments -A "-*" \
+            '(--tag --priority --mode )'{-h,--help}'[show this help message and exit]' \
+            '--tag=-[Also add repo to tag. Use tag::N to set priority]' \ 
+            '--priority=-[Set priority (when adding to tag)]' \ 
+            '--mode=-[Set merge mode]' \ 
+            '*: name [url]' \ 
+            && ret=0 
```

Comment:
I think the trailing whitespace causes an issue. Can we remove all trailing please.